test/e2e/specs/default-config/multi-form-input/multi-form-input.config.ts

```diff
@@ -3,9 +3,9 @@ import getConfig from '@isrd-isi-edu/chaise/test/e2e/setup/playwright.configurat
 export default getConfig({
   testName: 'default-config/multi-form-input',
   testMatch: [
-    'multi-form-input-clone.spec.ts',
+    // 'multi-form-input-clone.spec.ts',
     'multi-form-input-create.spec.ts',
-    'multi-form-input-edit.spec.ts'
+    // 'multi-form-input-edit.spec.ts'
```

Comment:
These 2 specs should be uncommented. Probably from when I noticed the permission issue in hatrac?

Reply:
Thank you for catching it. No I was debugging the `.clear()` changes that I did and forgot to revert it back before pushing.